docs/UserGuide.md

```diff
@@ -194,6 +206,18 @@ The following points explain the format of a command.
 * Duplicate company names are not allowed.<br>
   e.g. if there exists a company named `meta`, adding another company with the name `meta` is not allowed.
 
+<div markdown="block" class="alert alert-info">
+
+:information_source: Commands are recognised by their command word. As such, if the command word is incorrect, even if it differs slightly from the correct command word, Tinner's response box will indicate that you have entered an unknown command.
+
+</div>
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: Note that the invalid command messages in the response box will not specify the missing parameters in the command.
```

Comment:
Could change our application to account for this in the future, but good to keep it explicit here for now 👍